src/os/os2/os2.cpp

```diff
@@ -76,11 +76,11 @@ void FiosGetDrives(FileList &file_list)
 			fios->type = FIOS_TYPE_DRIVE;
 			fios->mtime = 0;
 #ifndef __INNOTEK_LIBC__
-			snprintf(fios->name, lengthof(fios->name),  "%c:", 'A' + disk - 1);
+			fios->name = fmt::format("{:c}:", 'A' + disk - 1);
```

Comment:
Can we get away without fmt here?
`fios->name = std::string(2, 'A' + disk - 1); fios->name[1] = ':';` ? Bit clunky, perhaps.

Similarly for other places

Reply:
I've gone for a slightly different approacn, which should be fine allocation wise given the short string optimisations.